src/OptionSettlementEngine.sol

```diff
@@ -75,6 +78,19 @@ contract OptionSettlementEngine is ERC1155, IOptionSettlementEngine {
     /// @notice Accessor for mapping a claim id to its ClaimIndices
     mapping(uint256 => OptionLotClaimIndex[]) internal _claimIdToClaimIndexArray;
 
+    /*//////////////////////////////////////////////////////////////
+    //  Modifiers
+    //////////////////////////////////////////////////////////////*/
+
+    /// @notice This modifier restricts function access to the feeTo address.
+    modifier onlyFeeTo() {
```

Comment:
Because we now have 3 permissioned functions, I refactored to extract a modifier. But might we go with solmate `Owned` now, because we only have 1 role?